bftengine/include/bftengine/Replica.hpp

```diff
@@ -50,20 +55,28 @@ enum MsgFlag : uint8_t {
 // 2. Key exchange
 // 3. Change DB scheme
 // and basically any management action that is handled by the layer that uses concord-bft.
-class ControlHandlers {
+class IControlHandler {
```

Comment:
We have a separate `ControlHandler.hpp`. Would you mind adding this class to the same file or create a new one?

Also, do we need this interface class? Why don't we just use a ControlHandler? Do we foresee in the near future that we have multiple ControlHandler implementations?

Reply:
The reason why IControlHandler is part of Replica.hpp is because it is part of the bftengine interfaces.
A user can load into the singleton a custom implementation that is relevant for his own application. Note that ControlHandler.hpp is part of the src directory of bftengine.

We decided to have an interface class to enlarge the flexibility.
This way, if we want to change in concord the way we handle the events in concordbft, we won't need to change the default implementation in cocordbft